test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -180,6 +180,13 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.apm.managedServiceUrl (any)',
         'xpack.apm.serverlessOnboarding (any)',
         'xpack.apm.latestAgentVersionsUrl (string)',
+        'xpack.apm.featureFlags.agentConfigurationAvailable (any)',
+        'xpack.apm.featureFlags.configurableIndicesAvailable (any)',
+        'xpack.apm.featureFlags.infrastructureTabAvailable (any)',
+        'xpack.apm.featureFlags.infraUiAvailable (any)',
+        'xpack.apm.featureFlags.migrationToFleetAvailable (any)',
+        'xpack.apm.featureFlags.sourcemapApiAvailable (any)',
+        'xpack.apm.featureFlags.storageExplorerAvailable (any)',
```

Comment:
hi @thomheymann,
I set them initially to boolean but the test were failing
<img width="1046" alt="image" src="https://github.com/elastic/kibana/assets/31922082/e0f5a975-32e2-446d-911a-a2b1cdb374e0">

Reply:
That's strange. Maybe the renderer doesn't infer the type correctly because of the context conditional:
https://github.com/elastic/kibana/pull/159136/files#diff-fa8fe01d402a7eb5cbc387759a4ea3930f3cc056d90477c04232d3ab91afaac7R17-R25
Let's leave as is for now in that case.

Reply:
Yes if we use a conditional rendering, the renderer is currently unable to infer the type based on provided default value. Hence it will always result in `any` type